site/docs/guides/create-derivation.md

```diff
@@ -0,0 +1,251 @@
+---
+sidebar_position: 2
+---
+# Create a derivation with flowctl
+
+Once you're familiar with creating a basic [Data Flow](../concepts/README.md#essential-concepts), you can take things a step further
+and transform your data with [derivations](../concepts/derivations.md).
+A **derivation** is a kind of Flow collection that results from the transformation of one or more other collections.
+
+:::info Beta
+Derivation creation is currently a developer workflow that uses [flowctl](../concepts/flowctl.md).
+Support for derivations in the Flow web application will be added in the future.
+:::
+
+## Prerequisites
+
+* An existing Flow **collection**. Typically, you create this through a **capture** in the Flow web application.
+If you need help, see the [guide to create a Data Flow](./create-dataflow.md).
+
+* **flowctl** installed locally. For help, see the [installation instructions](../concepts/flowctl.md#installation-and-setup).
+
+## Pull your specification files locally
+
+To begin working in your local environment, you must authenticate Flow from the command line.
+Then, you'll need to add your source collection's specification files to a **draft** and copy the draft locally.
```

Comment:
Should this be
> And ~copy~ **edit** that draft locally

?

Reply:
Ah, I see. Reading below, there is an explicit step to copy the draft locally. I would actually still word this as "edit" not "copy" (since the "copy" part is an implementation detail, whereas editing is what you actually care about), but this is a pretty nit-picky nit /shrug